src/ActionList/Item.tsx

```diff
@@ -190,7 +209,8 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       },
       '&:hover:not([aria-disabled]) + &, &:focus:not([aria-disabled]) + &, &[data-focus-visible-added] + li': {
         '--divider-color': 'transparent'
-      }
+      },
+      ...(active ? activeStyles : {})
```

Comment:
I'm not sure how to make TypeScript happy on [line 252](https://github.com/primer/react/pull/2064/files#diff-008c41e7105eb2bb0be2768425e46a21944dcbffc988ca984f91d3d11d280efdR252). @siddharthkp any ideas?

Reply:
Does `sx={merge<BetterSystemStyleObject>(styles, sxProp)}` help?